app/models/user.rb

```diff
@@ -1,18 +1,38 @@
 class User < ApplicationRecord
-  before_save :downcase_email
+  attr_accessor :remember_token
+  before_save{self.email = email.downcase}
```

Comment:
` email.downcase!` và `self.email = email.downcase` khác gì nhau ko em

Reply:
Em nghĩ là không . Nhưng vì muốn chắc chắn nên sử dụng self.email an toàn hơn :3